Documentation/Books/Manual/Administration/Configuration/README.md

```diff
@@ -0,0 +1,280 @@
+# Configuration
+
+The [programs and tools](../../Programs/README.md) shipped in an
+ArangoDB package can be configured with various _startup options_.
+
+- Startup options you specify on a command line are referred to as
+  [command line options](#command-line-options):
+
+  `arangosh --server.database myDB`
+
+- The same options can also be set via
+  [configuration files](#configuration-file-format),
+  using a slightly different syntax:
+
+  `server.database = myDB`
+
+- There are also _flags_ which are for command line usage only,
+  such as `--help` and `--version`. They don't take any value
+  in contrast to options.
+
+Find the available options and flags in the _Options_ sub-chapters of the
+respective [Programs & Tools](../../Programs/README.md) sub-chapter, like the
+[ArangoDB Server Options](../../Programs/Arangod/Options.md).
+
+The [ArangoDB Starter](../../Programs/Starter/README.md) works differently
+to the other programs and tools. It uses `setup.json` files for its own
+[configuration](../../Programs/Starter/Architecture.md#starter-data-directory)
+and has a fluent command line interface to execute certain actions.
+
+## Command line options
+
+Command line options can be supplied in the style `--option value` with two
+dashes (also known as hyphen minus), the name of the option, a space as
+separator and the value. You may also use an equals sign `=` as separator
+like `--option=value`.
+
+The value can be surrounded with double quote marks `"` like
+`--option="value"`. This is mandatory if the value contains spaces,
+but it is optional otherwise.
+
+Some binaries accept a positional argument, which means you can take a
+shortcut and leave out the `--option` part and supply the value directly.
+For _arangod_ it is the `--database.directory` option. The following
+commands are identical:
+
+```
+arangod my_data_dir
+arangod "my_data_dir"
+arangod --database.directory my_data_dir
+arangod --database.directory=my_data_dir
+arangod --database.directory "my_data_dir"
+arangod --database.directory="my_data_dir"
+```
+
+Many options belong to a section as in `--section.param`, e.g.
+`--server.database`, but there can also be options without any section.
+These options are referred to as _global options_.
+
+To list available options, you can run a binary with the `--help` flag:
+
+```
+arangosh --help
+```
+
+To list the options of a certain section only, use `--help-{section}`
+like `--help-server`. To list all options including hidden ones use
+`--help-.`.
```

Comment:
Or don't we want to tell users how to list the hidden options?

Reply:
@sleto-it You can decide here. As it is now, we should tell users to find out about some of these useful options as they are not documented in the online docs. We may remove it once we've unhidden all user-facing options, so that it will then list internal options only, which users should not use.